package.json

```diff
@@ -71,9 +71,12 @@
 		"@babel/eslint-plugin": "^7.13.16",
 		"@babel/preset-react": "^7.13.13",
 		"@cypress/code-coverage": "^3.9.6",
+		"@parcel/babel-plugin-transform-runtime": "^2.0.0-beta.3.1",
+		"@parcel/babel-preset-env": "^2.0.0-beta.3.1",
 		"@parcel/transformer-svg-react": "2.0.0-beta.3.1",
 		"@parcel/transformer-svgo": "2.0.0-beta.3.1",
 		"babel-plugin-istanbul": "^6.0.0",
+		"bootstrap": "^5.1.3",
```

Comment:
Waarvoor is bootstrap nodig?

Reply:
Voor de demo pagina (https://parleycdn.com/demo/v2/?accountId=&deviceId=&apiDomain=https%3A%2F%2Fapi.parley.nu&background=&language=nl&start=true)